cli/cli_fileshare.go

```diff
@@ -203,6 +208,28 @@ func (c *cmd) FileshareAutoCompletePeers(ctx *cli.Context) {
 	}
 }
 
+// FileshareAutoCompleteClear implements bash autocompletion for history clearing
+func (c *cmd) FileshareAutoCompleteClear(ctx *cli.Context) {
+	if ctx.NArg() == 0 {
+		fmt.Println("all\n<time_period>")
+	} else {
+		if ctx.Args().Get(0) == "all" {
+			return
+		}
+		last := ctx.Args().Get(ctx.Args().Len() - 1)
+		i, err := strconv.Atoi(last)
+		if err == nil {
+			if i == 1 {
+				fmt.Println("second\nminute\nhour\nday\nweek\nmonth\nyear")
+			} else {
+				fmt.Println("seconds\nminutes\nhours\ndays\nweeks\nmonths\nyears")
+			}
+		} else {
+			fmt.Println("<time_period>")
```

Comment:
if the first param is not an int, wouldn't be better not to print anything otherwise I think the autocomplete will display `<time_period`?

Reply:
It prints <time_period> if the previous (last) argument is a complete time period (e.g. ... 5d) or a unit (e.g. ... 5 days). This gives a hint to user that he can write next part of time span.

Reply:
same here, `\<time_period\>` is automatically added to the command when pressing tab.